fusor-ember-cli/app/components/progress-bar.js

```diff
@@ -5,25 +5,6 @@ export default Ember.Component.extend({
   tagName: 'div',
   classNames: ['row'],
 
-  runStartPolling: function() {
-    return this.send('startPolling');
-  }.on('didInsertElement'),
```

Comment:
removed `startPolling` out of components, since each of the 3 would call the model reload